images/hlf-k8s/bin/bootstrap.sh

```diff
@@ -77,36 +78,28 @@ function bootstrap() {
     kubectl exec $CA_POD_NAME -- bash -c "fabric-ca-client enroll -d --enrollment.profile tls -u http://$ADMIN_USERNAME:$ADMIN_PASSWORD@\$SERVICE_DNS:7054 -M $TLS_ADMIN_DIR --csr.hosts $CSR_HOSTS"
     kubectl exec $CA_POD_NAME -- bash -c "fabric-ca-client enroll -d --enrollment.profile tls -u http://$USER_USERNAME:$USER_PASSWORD@\$SERVICE_DNS:7054 -M $TLS_USER_DIR --csr.hosts $CSR_HOSTS"
 
+    # Fetch certificates/keys from CA pod
     kubectl cp $CA_POD_NAME:/var/hyperledger/fabric-ca/msp/certs/ /tmp/certs
-
     kubectl cp $CA_POD_NAME:$MSP_ADMIN_DIR /tmp/mspAdmin
     kubectl cp $CA_POD_NAME:$MSP_USER_DIR /tmp/mspUser
     kubectl cp $CA_POD_NAME:$TLS_ADMIN_DIR /tmp/tlsAdmin
     kubectl cp $CA_POD_NAME:$TLS_USER_DIR /tmp/tlsUser
 
-    mv /tmp/certs/*-cert.pem /tmp/certs/ca-cert.pem
-
+    # Normalize file names
+    mv /tmp/certs/*-cert.pem /tmp/certs/cacert.pem
     mv /tmp/mspAdmin/keystore/* /tmp/mspAdmin/keystore/key.pem
     mv /tmp/mspUser/keystore/* /tmp/mspUser/keystore/key.pem
-    mv /tmp/mspAdmin/cacerts/* /tmp/mspAdmin/cacerts/cacert.pem
-    mv /tmp/mspUser/cacerts/* /tmp/mspUser/cacerts/cacert.pem
```

Comment:
Indeed:
```
mv /tmp/mspAdmin/cacerts/* /tmp/mspAdmin/cacerts/cacert.pem	
mv /tmp/mspUser/cacerts/* /tmp/mspUser/cacerts/cacert.pem
```
is what we want to remove :)

Reply:
Do we want to support different CA Root certs for the MSP admin and the MSP user?

We currently have 4 possible CA root Certs:

- TLS server CA Root Cert
- TLS client CA Root Cert
- MSP admin CA Root Cert
- MSP user CA Root Cert

Which ones do we want to keep ? cc @Kelvin-M